src/Microsoft.AspNetCore.Mvc.Razor.Extensions/RazorPageDocumentClassifierPass.cs

```diff
@@ -10,6 +10,21 @@ namespace Microsoft.AspNetCore.Mvc.Razor.Extensions
     public class RazorPageDocumentClassifierPass : DocumentClassifierPassBase
     {
         public static readonly string RazorPageDocumentKind = "mvc.1.0.razor-page";
+        private static readonly RazorEngine LeadingDirectiveParsingEngine = RazorEngine.Create(builder =>
```

Comment:
Build a project engine instead, we aren't going to maintain the `RazorEngine.Create` overload.